src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -19,23 +19,31 @@ import '@polymer/iron-icon/iron-icon';
 import '@polymer/iron-icons/iron-icons';
 import './outline-step-view';
 
-import {css, customElement, html, LitElement, property, unsafeCSS} from 'lit-element';
+import {css, customElement, html, LitElement, property} from 'lit-element';
 
 import {COMMON_STYLES} from './cloud-install-styles';
+import {countryCode, CloudLocation, CountryCode} from '../../model/location';
+import {getShortName, localizeCountry} from '../location_formatting';
 
-export interface Location {
-  id: string;
-  name: string;
-  flag: string;
-  available: boolean;
-}
+// TODO: Add more flags
+const FLAG_IMAGE_DIR = 'images/flags';
+const FLAG_MAPPING: {[countryCode in CountryCode]?: string} = {
+  'IN': `${FLAG_IMAGE_DIR}/india.png`,
```

Comment:
This map is unfortunate. It would be nice to just name the flag files with the country code.
Though we still need to know when the flag is unavailable.

Reply:
Yes, the circle-flags library uses ISO country codes for the file names, so hopefully we'll be able to get rid of this map.